mailit/tests/plugin_test.py

```diff
@@ -109,11 +109,11 @@ def test_it_only_creates_templates_when_creating_not_when_updating(self):
 class MailSendingTestCase(TestCase):
     def setUp(self):
         super(MailSendingTestCase, self).setUp()
-        self.person3 = Person.objects.all()[2]
+        self.person3 = Person.objects.get(id=1)
```

Comment:
I've just checked and it doesn't matter, I think it is worth changing the name of the variable to just ```self.person``` or something simpler.